Cargo.toml

```diff
@@ -18,8 +18,15 @@ exclude = [
 ]
 
 # [patch.crates-io]
-# tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-rpc          = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-proto        = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-light-client = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-testgen      = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
+# tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "v0.23.x" }
+# tendermint-rpc          = { git = "https://github.com/informalsystems/tendermint-rs", branch = "v0.23.x" }
+# tendermint-proto        = { git = "https://github.com/informalsystems/tendermint-rs", branch = "v0.23.x" }
+# tendermint-light-client = { git = "https://github.com/informalsystems/tendermint-rs", branch = "v0.23.x" }
+# tendermint-testgen      = { git = "https://github.com/informalsystems/tendermint-rs", branch = "v0.23.x" }
+
+# [patch.crates-io]
+# tendermint              = { path = "../tendermint-rs/tendermint" }
+# tendermint-rpc          = { path = "../tendermint-rs/rpc" }
+# tendermint-proto        = { path = "../tendermint-rs/proto" }
+# tendermint-light-client = { path = "../tendermint-rs/light-client" }
+# tendermint-testgen      = { path = "../tendermint-rs/testgen" }
```

Comment:
Should we remove these?

Reply:
Yeah, this was left over from my work with a locally merged copy.